pydicom/dataset.py

```diff
@@ -1165,6 +1172,8 @@ def __setattr__(self, name, value):
                 # don't have this tag yet->create the data_element instance
                 VR = dictionary_VR(tag)
                 data_element = DataElement(tag, VR, value)
+                if VR == 'SQ':
```

Comment:
Can you add a comment on why this is here?

Reply:
Missed that - done now.